src/Entity/Unregistration.php

```diff
@@ -60,19 +60,26 @@ class Unregistration
      */
     private $unregisteredAt;
 
+    /**
+     * @ORM\Column(type="boolean", options={"default": false})
+     */
+    private $adherent = false;
```

Comment:
what about `isAdherent` (`is_adherent` in DB) for the column name too?
IMO it's clearer